integration-libs/opf/base/root/services/opf-resource-loader.service.ts

```diff
@@ -133,6 +133,9 @@ export class OpfResourceLoaderService extends ScriptLoader {
     if (html) {
       const element = new DOMParser().parseFromString(html, 'text/html');
       const script = element.getElementsByTagName('script');
+      if (!script?.[0]?.innerText) {
```

Comment:
Before, an exception was thrown when html did not have script tag which lead html snippet to not be displayed.
Html without 'script' tag is valid use case.